### ISSUE TITLE
Go Major does not work with private repos

### ISSUE THREAD

**Issue by KevinAnthony** [icholy/gomajor#18]
I'm looking at gomajor to evaluate when I need to update versions and for public repo's it's fine.  However for private repo's it does not list them.  
Is this something that is planned for the future.

My envvar GOPRIVATE is already set correctly.

**Comment by icholy** [comment]
Currently, everything matching `GOPRIVATE` is skipped:

https://github.com/icholy/gomajor/blob/c65b0b7389f4bf01dcb58c08888eb7eca8e09d98/main.go#L88-L90

Adding support for going directly to the VCS would require a significant amount of work that I can't commit to right now. But we can keep this issue open.

**Comment by sanyatuning** [comment]
Hi @icholy 
We use [Athens](https://docs.gomods.io/) proxy and a GOPROXY env to get private packages.
The proxy does not need authentication.
Can you update your package to use this env variable?

**Comment by icholy** [comment]
@sanyatuning yeah, can you create a separate issue. Something like "Support GOPROXY env".
